### PR TITLE
954: add chatlio live chat widget

### LINKED ISSUE #954: Integrere live-chat mellom nettside og slack
**Business requirement**
- Tilby en lavterskel kommunikasjonskanal som gjør det enklere for kunder og potensielle arbeidstakere å snakke med en Alv.

**User stories**
- Som kunde / arbeidstaker vil jeg enkelt kunne chatte direkte med en Alv gjennom Alv.no
- Som Alv vil jeg svare på chatmeldinger i Slack

Potensielle løsninger:
- https://chatlio.com/
- https://www.intercom.com/

### PR DESCRIPTION
**Pull request by Fordiboii**
Closes #954.

### CODE REVIEW

File: packages/mail-api/src/index.js [hunk 1]
```diff
@@ -12,4 +12,4 @@ app.use(bodyParser.urlencoded({ extended: true }));
 
 app.listen(port, () => {
```
Comment (klyngen) [inline packages/mail-api/src/index.js:13]: This is not relevant for this PR

**Review by klyngen — CHANGES_REQUESTED** [review]
Remove mail-api changes

**Comment by klyngen** [comment]
image
Build is failing. Run a local build to see if you get the same issues. The failing build is preventing you from getting a review-environment.

File: packages/mail-api/src/index.js (hunk 1 above)
Comment (Fordiboii) [reply packages/mail-api/src/index.js:13]: Had to fix it to allow committing

**Comment by klyngen** [comment]
💻 https://954-live-c-955.review.test-alv.no

**Review by klyngen — APPROVED** [review]

**Pull request merged**